router/router.go

```diff
@@ -1748,7 +1787,7 @@ func (rt *HandleT) readAndProcess() int {
 
 	rt.lastQueryRunTime = time.Now()
 
-	pickupMap, latenciesUsed := rt.MultitenantI.GetRouterPickupJobs(rt.destName, rt.noOfWorkers, timeOut, jobQueryBatchSize, rt.timeGained)
+	pickupMap, _ := rt.MultitenantI.GetRouterPickupJobs(rt.destName, rt.noOfWorkers, timeOut, jobQueryBatchSize, rt.timeGained)
```

Comment:
@cisse21 `latenciesUsed` doesn't appear to be needed anymore after removing job draining from pickupLoop. Please confirm this looks good